client/src/pages/private/ParticipantEOI.js

```diff
@@ -0,0 +1,240 @@
+import React, { useEffect, useState } from 'react';
+import { withStyles, makeStyles } from '@material-ui/core/styles';
+import { Typography, Grid, Button } from '@material-ui/core';
+import { Redirect, useParams, useLocation, useHistory } from 'react-router-dom';
+import { red } from '@material-ui/core/colors';
+import LinearProgress from '@material-ui/core/LinearProgress';
+import DialogActions from '@material-ui/core/DialogActions';
+import DialogContent from '@material-ui/core/DialogContent';
+import store from 'store';
+import { Routes, ToastStatus } from '../../constants';
+
+import { Page, Alert } from '../../components/generic';
+import { Form } from '../../components/participant-form';
+import { API_URL } from '../../constants';
+import { useToast } from '../../hooks';
+import { Dialog } from '../../components/generic';
+
+const rootUrl = `${API_URL}/api/v1/participant-user/participant`;
+
+// Custom UI
+const DeleteButton = withStyles((theme) => ({
+  root: {
+    color: theme.palette.getContrastText(red[500]),
+    backgroundColor: red[500],
+    '&:hover': {
+      backgroundColor: red[700],
+    },
+  },
+}))(Button);
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    width: '100%',
+    '& > * + *': {
+      marginTop: theme.spacing(2),
+    },
+  },
+}));
+
+// Helper methods
+const fetchParticipant = async (id) => {
+  try {
+    const response = await fetch(`${rootUrl}/${id}`, {
+      method: 'GET',
+      headers: {
+        Authorization: `Bearer ${store.get('TOKEN')}`,
+        Accept: 'application/json',
+        'Content-type': 'application/json',
+      },
+    });
+    const participants = await response.json();
```

Comment:
```suggestion
    const [participants] = await response.json();
    return participants;
```
would this work? It feels a bit cleaner to me

Reply:
100% works, I will change this